### PR TITLE
You must confirm elements deletion.

### PR DESCRIPTION
**Pull request by CezaryH257**
If you want to delete elements (such as class, interface etc.), you must confirm that deletion by clicking yes or no in confirmation window.

### CODE REVIEW

File: violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java [hunk 1]
```diff
@@ -121,23 +144,32 @@ public IGraph getGraph()
...
+            {
+                this.selectionHandler.clearSelection();
+                this.behaviorManager.fireAfterRemovingSelectedElements();
+            }
+    	}
     }
 
+    private boolean isUserConfirmedElementsDeletion()
+    {
+    	int reply = JOptionPane.showConfirmDialog(null, deleteDialogText, null, JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE);
```
Comment (NdV66) [inline violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:169]: maybe 
return JOptionPane.showConfirmDialog(null, deleteDialogText, null, JOptionPane.YES_NO_OPTION, JOptionPane.WARNING_MESSAGE) == JOptionPane.YES_NO_OPTION

**Review by NdV66 — COMMENTED** [review]
Fix or reply

File: violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java [hunk 2]
```diff
@@ -47,7 +49,28 @@
...
+	private int lastHeight = 0;
+	
+	/**
+	 * Scale factor used to grow drawing area
+     */
+	private static final double GROW_SCALE_FACTOR = Math.sqrt(2);
+	
+	private IEditorPartBehaviorManager behaviorManager = new EditorPartBehaviorManager();
+	
+    private String deleteDialogText = "Delete selected elements?";
```
Comment (abizukojc) [inline violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:71]: Why is this hard-coded ? You should use properties for all languages available in VioletUML

Comment (orchowski) [reply violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java:71]: I agree with that

**Review by RoughTomato — CHANGES_REQUESTED** [review]
Fix already commented functionalities

**Comment by orchowski** [comment]
Looks good to me. I didn't have time to review the changes. But I see that it's not too late :)

**Review by orchowski — APPROVED** [review]

**Comment by Peroniada** [comment]
Please squash commits to make it eligible for merge

**Review by Peroniada — APPROVED** [review]

**Pull request closed without merging**